examples/air/air.py

```diff
@@ -233,7 +233,7 @@ def guide_step(self, t, n, prev, inputs):
         # Sample presence.
         z_pres = pyro.sample('z_pres_{}'.format(t),
                              dist.Bernoulli(z_pres_p * prev.z_pres).independent(1),
-                             infer=dict(baseline=dict(baseline_value=bl_value.squeeze(-1))))
+                             infer=bl_value)
```

Comment:
Can you rename this variable from `bl_value` to `infer_dict` to clarify that it is a configuration dict rather than a tensor?